src/app/search/loadouts/search-filters/freeform.ts

```diff
@@ -33,8 +45,10 @@ const freeformFilters: FilterDefinition<
     keywords: ['name', 'exactname'],
     description: tl('LoadoutFilter.Name'),
     format: 'freeform',
-    suggestionsGenerator: ({ loadouts }) =>
-      loadouts?.map((loadout) => `exactname:${quoteFilterString(loadout.name.toLowerCase())}`),
+    suggestionsGenerator: ({ loadouts, selectedLoadoutsStore }) =>
+      loadouts
+        ?.filter((loadout) => isLoadoutCompatibleWithStore(loadout, selectedLoadoutsStore))
```

Comment:
In the end I decided to not lift this, we still need it in this file to use in the filter functions so it felt cleaner to just hold all the logic here.